src/content/1.7/development/architecture/migration-guide/forms/settings-forms.md

```diff
@@ -56,22 +56,20 @@ As an example, here's how the Administration page's Form Handler service is decl
 ```yaml
 # /src/PrestaShopBundle/Resources/config/services/form/form_handler.yml
 
-prestashop.adapter.administration.form_handler:
-  class: 'PrestaShop\PrestaShop\Core\Form\FormHandler'
-  arguments:
-    - '@=service("form.factory").createBuilder()'
-    - '@prestashop.core.hook.dispatcher'
-    - '@prestashop.adapter.administration.form_provider'
-    -
-      'general': 'PrestaShopBundle\Form\Admin\AdvancedParameters\Administration\GeneralType'
-      'upload_quota': 'PrestaShopBundle\Form\Admin\AdvancedParameters\Administration\UploadQuotaType'
-      'notifications': 'PrestaShopBundle\Form\Admin\AdvancedParameters\Administration\NotificationsType'
-    - 'AdministrationPage'
+prestashop.adapter.administration.general.form_handler:
+    class: 'PrestaShop\PrestaShop\Core\Form\Handler'
```

Comment:
This class is new in 1.7.8, perhaps it should be noted

Reply:
Done